### PR TITLE
Tolowercase

### PR DESCRIPTION
**Pull request by M3RF3R**
### Description

Creation of the folder toLowerCase and the toLowerCase file to explain this Java method
### Type of Change

- Adding a new entry

### Checklist

- [X] All writings are my own.
- [X] My entry follows the Codecademy Docs style guide.
- [X] My changes generate no new warnings.
- [X] I have performed a self-review of my own writing and code.
- [X] I have checked my entry and corrected any misspellings.
- [X] I have made corresponding changes to the documentation if needed.
- [X] I have confirmed my changes are not being pushed from my forked `main` branch.
- [X] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [X] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@M3RF3R I did a first review and left a few comments. The most important is to remove the style.md file from this branch to avoid a conflict.

File: content/html/concepts/attributes/terms/style/style.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
+---
```
[the remark this answers is not in the document]
Comment (M3RF3R) [reply content/html/concepts/attributes/terms/style/style.md]: done

File: content/java/concepts/strings/terms/toLoweCase/toLowerCase.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+## Syntax
+
+```java
+public String toLowerCase();
+```
+
+## Example 
+
+Converting uppercase characters to lowercase characters:
```
[the remark this answers is not in the document]
Comment (M3RF3R) [reply content/java/concepts/strings/terms/toLoweCase/toLowerCase.md:27]: done

File: content/java/concepts/strings/terms/toLoweCase/toLowerCase.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+  - 'paths/computer-science'
+---
+
+The **`.toLowerCase()`** method converts all the characters of a string to lowercase characters in Java.
+
+> **Note:** This method is locale sensitive, in consequence it might produce unexpected results because it takes into account the user's locale, particularly when the string represents locale-specific values, e.g. date or time.
+
+## Syntax
+
+```java
```
[the remark this answers is not in the document]
Comment (M3RF3R) [reply content/java/concepts/strings/terms/toLoweCase/toLowerCase.md:21]: Hi! Thanks for your revision. I'm not sure I understand this comment. Everywhere I look they have this as syntax for the method: public String toLowerCase(). Should I just copy your suggestion (lowerCaseString = upperCaseString.toLowerCase();) and say that `lowerCaseString` stands for a string with lowercase characters and 'upperCaseString` stands for a string with some uppercase characters that will be converted?

**Comment by M3RF3R** [comment]
@SSwiniarski Hi! I've left you a comment, since I need some clarification. Thanks!

File: content/java/concepts/strings/terms/toLoweCase/toLowerCase.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (SSwiniarski) [reply content/java/concepts/strings/terms/toLoweCase/toLowerCase.md:21]: @M3RF3R yes, if you could do that please.

[the remark this answers is not in the document]
Comment (M3RF3R) [reply content/java/concepts/strings/terms/toLoweCase/toLowerCase.md:21]: Is this correct?

**Review by SSwiniarski — APPROVED** [review]
@M3RF3R Looks good for a second review.

File: content/java/concepts/strings/terms/toLoweCase/toLowerCase.md [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+Title: '.toLowerCase()'
+Description: 'Converts a string to lowercase characters.'
+Subjects:
+  - 'Computer Science'
+Tags:
+  - 'Characters'
+  - 'Strings'
+  - 'Methods'
+CatalogContent:
+  - 'learn java'
```
Comment (caupolicandiaz) [inline content/java/concepts/strings/terms/toLoweCase/toLowerCase.md:11]:
```suggestion
  - 'learn-java'
```

File: content/java/concepts/strings/terms/toLoweCase/toLowerCase.md [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+---
+
+The **`.toLowerCase()`** method converts all the characters of a string to lowercase characters in Java.
+
+> **Note:** This method is locale sensitive, in consequence it might produce unexpected results because it takes into account the user's locale, particularly when the string represents locale-specific values, e.g. date or time.
+
+## Syntax
+
+```java
+lowerCaseString = upperCaseString.toLowerCase();
```
Comment (caupolicandiaz) [inline content/java/concepts/strings/terms/toLoweCase/toLowerCase.md:22]:
```suggestion
upperCaseString.toLowerCase();
```

File: content/java/concepts/strings/terms/toLoweCase/toLowerCase.md [hunk 6]
```diff
@@ -0,0 +1,42 @@
...
+```
+Where `lowerCaseString` is the string with all lower case characters and `upperCaseString` is the string with some or all upper case characters that will be converted.
```
Comment (caupolicandiaz) [inline content/java/concepts/strings/terms/toLoweCase/toLowerCase.md:24]:
```suggestion
Where `upperCaseString` is the string with some or all upper case characters to be converted.
```

File: content/java/concepts/strings/terms/toLoweCase/toLowerCase.md [hunk 7]
```diff
@@ -0,0 +1,42 @@
...
+
+## Example 
+
+This example converts uppercase characters to lowercase characters:
+
+```java
```
Comment (caupolicandiaz) [inline content/java/concepts/strings/terms/toLoweCase/toLowerCase.md:30]:
```suggestion
```java
// Example.java

```

**Review by caupolicandiaz — CHANGES_REQUESTED** [review]
@M3RF3R Just a few minor changes

File: content/java/concepts/strings/terms/toLoweCase/toLowerCase.md (hunk 4 above)
Comment (M3RF3R) [reply content/java/concepts/strings/terms/toLoweCase/toLowerCase.md:11]: done

File: content/java/concepts/strings/terms/toLoweCase/toLowerCase.md (hunk 6 above)
Comment (M3RF3R) [reply content/java/concepts/strings/terms/toLoweCase/toLowerCase.md:24]: done

File: content/java/concepts/strings/terms/toLoweCase/toLowerCase.md (hunk 7 above)
Comment (M3RF3R) [reply content/java/concepts/strings/terms/toLoweCase/toLowerCase.md:30]: done

**Comment by M3RF3R** [comment]
@caupolicandiaz I added the changes!

**Review by caupolicandiaz — APPROVED** [review]
@M3RF3R Looks good

**Comment by SSwiniarski** [comment]
Congrats on the new entry @M3RF3R. The live entry can be found here: https://www.codecademy.com/resources/docs/java/strings/toLowerCase

It looks as if you're anonymous:
image

If you want to be credited follow the instructions below to add your GitHub username to your Codecademy profile. If not, there is nothing more you need to do.

To add your GitHub username to your Codecademy profile:

1.) Add your GitHub Username to your Codecademy Profile Details, you can get there on your desktop by pressing on your profile picture in the upper right-hand corner and selecting "Profile" in the drop-down box:

image

2.) Then log out and log back into Codecademy using your GitHub login:

image

**Pull request merged**